tests/automated/sys/net/gnrc/network_layer/ipv6/suite-01_init/Makefile

```diff
@@ -0,0 +1,18 @@
+APPLICATION = ipv6
+#include ../Makefile.tests_common
+BOARD ?= native
+RIOTBASE ?= $(CURDIR)/../../../../../../../../
```

Comment:
> Writing that line, I was curious if someone may show me that RIOTs make system provides a better method to examine the project root.

The problem here is, that this line is called before the RIOT Make system is actually invoked. You could put the Makefile to build the different tests directly in the tests folder (similar to the unittests) and make building and executing of the tests callable from the common tests folder instead of from the application itself.